tests/piqueserver/test_extensions.py

```diff
@@ -4,12 +4,22 @@
 import os
 
 from twisted.trial import unittest
+
 from piqueserver import extensions
 from piqueserver.config import config
-# from piqueserver.server import scripts_option
-scripts_option = config.option('test_scripts', default=[], validate=extensions.check_scripts)
+from piqueserver.server import FeatureProtocol
+from piqueserver.player import FeatureConnection
 
 class TestExtensions(unittest.TestCase):
+    # Define scripts
+    script_names = ['test_script_game_mode', 'test_script_regular']
+    script_names_length = len(script_names)
+    # Get current directory
+    curr_dir = os.getcwd()
+    script_dir = os.path.join(curr_dir, 'piqueserver/test_scripts/')
```

Comment:
sorry, I didn't see this. The test scripts should go into the `tests/` directory, not the `piqueserver/` directory, so that it does not appear in the released packages and automatic module documentation.

Reply:
basically: `path.join(path.dirname(__file__), "test_scripts/")`